docs/.vuepress/config.ts

```diff
@@ -276,6 +281,24 @@ module.exports = defineConfig({
                     faq: "Часто Задаваемые Вопросы",
                 }),
             },
+            "/uk-UA/": {
+                // text for the language dropdown
+                selectText: "Переклади",
+                // label for this locale in the language dropdown
+                label: "Українська",
+                // Custom text for edit link. Defaults to "Edit this page"
+                editLinkText: "Edit this page on GitHub",
+                // Custom navbar values
+                nav: [{ text: "Налаштування", link: "/uk-UA/config/" }],
+                // Custom sidebar values
+                sidebar: sidebar("uk-UA", {
+                    guide: "Guide",
```

Comment:
@Andygol as well as this one.

I've extracted the other strings from elsewhere in the translations.